### PR TITLE
[13.0][MIG] stock_request

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Standard migration

CC @Forgeflow @etobella

### CODE REVIEW

File: stock_request/__manifest__.py [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 
 {
     "name": "Stock Request",
     "summary": "Internal request for stock",
-    "version": "12.0.1.1.6",
+    "version": "13.0.1.0.0",
     "license": "LGPL-3",
     "website": "https://github.com/stock-logistics-warehouse",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (AdriaGForgeFlow) [inline stock_request/__manifest__.py]: Could you change this to ForgeFlow?

**Review by AdriaGForgeFlow — CHANGES_REQUESTED** [review]
Quantities are not correctly updated in the Stock Request

**Comment by HviorForgeFlow** [comment]
> Quantities are not correctly updated in the Stock Request

It works on runbot for me. Can you check to reproduce it there? Thanks,

**Review by JoanSForgeFlow — APPROVED** [review]
Functional review, LGTM :+1:

**Review by ps-tubtim — APPROVED** [review]
Code LGTM. Functional test: OK.

**Comment by LoisRForgeFlow** [comment]
@AdriaGForgeFlow Your comments will be fixed with stock_request_purchase.

Ready to merge: 

/ocabot merge

**Pull request merged**